src/pages/management/configuration-api/index.mdx

```diff
@@ -3256,11 +3256,12 @@ Resource limits in the LiveChat product are different for each plan. This method
 
 |                                  |                                                                                       |
 | -------------------------------- | ------------------------------------------------------------------------------------- |
-| HTTP Method                      | **GET**                                                                               |
```

Comment:
shouldn't we leave HTTP Method: POST here?

Reply:
I deleted the HTTP method completely, as we never have it with POST methods, please see example [here](https://developers.livechat.com/docs/management/configuration-api#list-channels)